src/modules/claim/claim.dto.spec.ts

```diff
@@ -95,7 +96,14 @@ describe('ClaimRequestDTO', () => {
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
     claimRequest.registrationTypes = ['NotRealRegistrationType'];
-    await expect(ClaimRequestDTO.create(claimRequest)).rejects.toThrow;
+    try {
+      // It should throw exception
+      await ClaimRequestDTO.create(claimRequest);
+      expect(true).toBeFalsy();
+    } catch (err) {
+      expect(err).toHaveLength(1);
+      expect(err[0]).toBeInstanceOf(ValidationError);
+    }
```

Comment:
though this is technically correct, we prefer to use the much more cleaner rejects.Throw provided by jest.